g3/config/openai.py

```diff
@@ -7,7 +7,7 @@
 
 class OpenAIConfig(BaseSettings):
     api_key: str = ""
-    api_type: Optional[str] = "openai"
+    api_type: Optional[str] = "open_ai"
```

Comment:
@Workable/team-g change your configured `api_type` in your `$HOME/.g3/config`. Openai SDK defaults to `open_ai` instead of `openai`.